boards/adi/max78002evkit/max78002evkit_max78002_m4.dts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <adi/max32/max78002.dtsi>
+#include <adi/max32/max78002-pinctrl.dtsi>
+#include <zephyr/dt-bindings/gpio/adi-max32-gpio.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	 model = "Analog Devices MAX78002EVKIT";
+	 compatible = "adi,max78002evkit";
+
+	 chosen {
+		 zephyr,console = &uart0;
+		 zephyr,shell-uart = &uart0;
+		 zephyr,sram = &sram2;
+		 zephyr,flash = &flash0;
+	 };
+
+	 leds {
+		 compatible = "gpio-leds";
+		 led1: led_1 {
+			 gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
+			 label = "Green LED";
+		 };
+		 led2: led_2 {
+			 gpios = <&gpio2 5 GPIO_ACTIVE_HIGH>;
+			 label = "Red LED";
+		 };
+	 };
+
+	 buttons {
+		 compatible = "gpio-keys";
+		 pb1: pb1 {
+			 gpios = <&gpio2 6 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			 label = "SW2";
+			 zephyr,code = <INPUT_KEY_0>;
+		 };
+		 pb2: pb2 {
+			 gpios = <&gpio2 7 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			 label = "SW3";
+			 zephyr,code = <INPUT_KEY_1>;
+		 };
+		 pb_wakeup: pb_wakeup {
+			gpios = <&gpio3 1 (GPIO_PULL_UP | GPIO_ACTIVE_LOW
+								| MAX32_GPIO_VSEL_VDDIOH)>;
```

Comment:
indent looks suspicious

Reply:
It's normal in code editor. I think it's about editors' tab view differences.

Reply:
Zephyr tab is 8 spaces, the the spacing shown on github is correct, replaced with spaces would be:
```
                 pb_wakeup: pb_wakeup {
                        gpios = <&gpio3 1 (GPIO_PULL_UP | GPIO_ACTIVE_LOW
                                                                | MAX32_GPIO_VSEL_VDDIOH)>;
```

Reply:
Updated, thanks.